app/src/main/res/values-th/strings.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE resources [
+	<!ENTITY appname "Slideshow">
+	<!ENTITY author "Michael Standen">
+	]>
+
+<resources>
+	<string name="app_name">&appname;</string>
+	<string name="action_settings">การตั้งค่า</string>
+	<string name="action_credits">เกี่ยวกับ</string>
+	<string name="action_picture_in_picture">Picture in Picture</string>
+
+	<string name="no_picture_in_picture">อุปกรณ์ของคุณไม่รองรับ Picture in picture</string>
+
+	<!-- List -->
+	<string name="inaccessible">(inaccessible)</string>
+	<string name="go_home_folder">ติดปัญหา? กดเพื่อกลับไปหน้าหลัก</string>
+	<string name="play_folder">เริ่มเล่นสไลด์จากตรงนี้</string>
+
+	<!-- Slideshow -->
+	<string name="title_activity_image">&appname;</string>
+
+	<string name="delete_button">ลบ</string>
+	<string name="delete_dialog_title">ลบภาพ</string>
+	<string name="delete_dialog_message">คุณแน่ใจหรือไม่?</string>
+	<string name="image_deleted">รูปถูกลบแล้ว</string>
+	<string name="image_not_deleted">ไม่สามารถลบรูปได้</string>
+
+	<string name="share_button">แบ่งปัน</string>
+	<string name="share_via">แบ่งปันผ่าน&#8230;</string>
+
+	<string name="image_detail_dimensions">มิติ: %1$dx%2$d</string>
+	<string name="image_detail_size">ขนาดไฟล์: %1$s</string>
+	<string name="image_detail_modified">แก้ไขล่าสุดเมื่อ: %1$s</string>
+
+	<string name="toast_no_files">ไม่พบไฟล์</string>
+
+	<string name="long_loading_warning">%1$s ใช้เวลานานในการโหลด</string>
+	<string name="long_loading_skipping">%1$s ใช้เวลานานในการโหลด กำลังข้ามรูป</string>
+	<string name="long_loading_skip_action">ข้ามรูป</string>
+
+	<!-- Image descriptions -->
+	<string name="image_description">รูปภาพ</string>
+	<string name="logo_description">โลโก้</string>
+
+	<!-- Credits -->
+	<string name="title_activity_credits">เกี่ยวกับ &appname;</string>
+	<string name="credits_creator">สร้างโดย <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">Slideshow คือแอพพลิเคชั่นดูรูปง่ายๆ บนมือถือ</string>
+	<string name="credits_content2">แบบเรียบง่ายและพื้นฐานมากที่สุดในตลาด โดยที่ไม่ต้องมีพื้นฐานมาก่อน
+	</string>
+	<string name="credits_content3">คุณสามารถดูซอร์สโค้ด หรือรายงานบัค หรือขอฟีเจอร์ได้ที่
+		<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>.
+	</string>
+	<string name="credits_content4">แอพพลิเคชั่นนี้ฟรี และ open source
+		ภายใต้ <a href="https://tldrlegal.com/license/mit-license">MIT license</a>.
+	</string>
+	<string name="credits_content5">ขอบคุณ <a href="https://github.com/psyanite">Yan Tsui</a> ที่ช่วยทดสอบและออกแบบไอคอนให้กับ &appname;
+	</string>
+	<string name="credits_content6">&appname;  แปลไทยโดย <a href="https://michael.standen.link">&author;</a>.
```

Comment:
Please replace the link with **your** name and website (or Github link). You did the translation so you get to be in the credits! :)